src/FileList/FileListHandler.cc

```diff
@@ -12,6 +12,7 @@
 #include <casacore/casa/OS/File.h>
 
 #include "FileInfoLoader.h"
+#include "Logger/Logger.h"
```

Comment:
In places like this I don't think we should be importing the logger if we only want to use fmt.